Cargo.toml

```diff
@@ -1,13 +1,15 @@
 [workspace]
 members = [
-    "hotham",
-    "hotham-simulator",
-    "examples/shared",
-    "examples/simple-scene",
-    "examples/complex-scene",
-    "examples/custom-rendering",
```

Comment:
Custom rendering example was removed by mistake here.

Reply:
And doesn't compile if added...

Reply:
Hrm, let me fix that up.